libs/openFrameworksCompiled/project/osx/CoreOF.xcconfig

```diff
@@ -1,3 +1,19 @@
+ARCHS = $(ARCHS_STANDARD_32_64_BIT)	// this allows you to select target architecture in dropdown menu
+VALID_ARCHS = x86_64 i386
+
+ONLY_ACTIVE_ARCH = YES      // this makes sure not to build an universal binary
```

Comment:
Hmm, CoreOF.xcconfig is also used by the OF Xcode project, doesn't this disallow fat libs across the board and not just when building Debug?